server/fleet/app.go

```diff
@@ -293,6 +293,10 @@ type ListOptions struct {
 	After string `query:"after,optional"`
 }
 
+func (l ListOptions) Empty() bool {
+	return l.Page == 0 && l.PerPage == 0 && l.OrderKey == "" && l.OrderDirection == 0 && l.MatchQuery == ""
```

Comment:
I don't know if it's deliberate or an omission, but you didn't check for `l.After`. If you want to check all fields (which I think it should, otherwise it's potentially confusing), it would be simpler to just do `return l == ListOptions{}`.

Reply:
I moved this method and did not make any changes to it.

Reply:
Yeah, I'm not sure what the reason was here. It looks like the `After` field was added later and we forgot to update the `Empty` method. This change should protect against that, but it won't work if we add non-basic fields eg slices.